src/gui/org/deidentifier/arx/gui/view/impl/menu/DialogOrderSelection.java

```diff
@@ -303,19 +303,42 @@ protected void createButtonsForButtonBar(final Composite parent) {
             public void widgetSelected(final SelectionEvent e) {
                 String file = controller.actionShowOpenFileDialog(getShell(), "*.csv"); //$NON-NLS-1$
                 if (file != null){
-                    String[] array = loadArray(file, Charset.defaultCharset());
-                    if (array != null) {
-                        
-                        // Select string
-                        for (int i=0; i<combo.getItems().length; i++){
-                            if (combo.getItem(i).equals("String")) { //$NON-NLS-1$
-                                combo.select(i);
+                    ArrayList<String> fileData = loadFile(file, Charset.defaultCharset());

```

Comment:
It seems to be possible that loadFile() returns "null". You need to either change "loadFile" or check for "null" here.